src/main/java/org/sagebionetworks/web/server/servlet/DiscussionMessageServlet.java

```diff
@@ -60,13 +60,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 	 *
 	 * @return
 	 */
-	private SynapseClient createNewClient(String sessionToken) {
+	private SynapseClient createNewClient(String accessToken) {
```

Comment:
You'll see many of these too (for some reason this is reimplemented over and over again).  Rename sessionToken to accessToken, use the SynapseClient.setBearerAuthorizationToken() instead of setSessionToken(), and only set if accessToken is not null.